src/Orchard.Environment.Extensions/ExtensionLibraryService.cs

```diff
@@ -508,17 +490,17 @@ private bool IsAmbientAssembly(string assemblyName)
 
         private bool IsAssemblyLoaded(string assemblyName)
         {
-            bool loaded;
-            return _loadedAssemblies.TryGetValue(assemblyName, out loaded);
+            Lazy<Assembly> assembly;
+            return _loadedAssemblies.TryGetValue(assemblyName, out assembly);
```

Comment:
Why did not use _loadedAssemblies.ContainsKey instead of TryGetValue
```
 public bool IsAssemblyLoaded(string assemblyName)
        {
            return _loadedAssemblies.ContainsKey(assemblyName);
        }
```